docs/system_overview/configuration.md

```diff
@@ -923,3 +923,52 @@ $ iadmin get_grid_configuration authentication password_max_time
 ```
 
 See [set_grid_configuration](../../icommands/administrator/#set_grid_configuration) and [get_grid_configuration](../../icommands/administrator/#get_grid_configuration) for more details about these subcommands.
+
+## Managing Parallel Transfer Threads
+
+When transferring large amounts of data between the client and server or even between servers, the number of threads used to read and write data can be configured by the administrator.
+
+First, we will go over how to configure what "large amounts of data" means. This is a configurable value that is controlled by the `maximum_size_for_single_buffer_in_megabytes` setting in `server_config.json`. This setting controls the maximum size of a particular file or data object that will be put into an in-memory buffer. If the size of the file or data object to read or write exceeds this size, iRODS will initiate what is known as a "parallel transfer".
+
+In a parallel transfer, a certain number of threads will be instantiated and tasked with reading and/or writing a number of bytes from a source file or data object to a destination file or data object. The number of transfer threads can be manipulated by the client and the server.
+
+Note: If the number of transfer threads is set to 0, this is a special "streaming" mode in which bytes are read and written via a single stream.
+
+### Client: `numThreads` keyword
+
+The way that the client can request the number of threads to use in parallel transfer is via the `numThreads` keyword. For some iCommands, this is provided using the `-N` option. From the [`iput` help text](../../icommands/user/#iput):
+```
+ -N  numThreads - the number of threads to use for the transfer. A value of
+       0 means no threading. By default (-N option not used) the server 
+       decides the number of threads to use.
+```
+
+### Legacy Static Policy Enforcement Point - `acSetNumThreads`
```

Comment:
Does this need "Server: " added to the beginning like the next section?

Reply:
Done